src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -322,6 +336,10 @@ private ZipFileProcessingResult processZipFileContent(
             log.warn("Rejected file {} from container {} - failed previously", zipFilename, containerName, ex);
             handleEventRelatedError(Event.DOC_UPLOAD_FAILURE, containerName, zipFilename, ex);
             return null;
+        } catch (ZipFileLoadException ex) {
+            log.warn("Rejected file {} from container {} - failed to load", zipFilename, containerName, ex);
+            handleEventRelatedError(Event.ZIPFILE_LOAD_FAILURE, containerName, zipFilename, ex);
+            return null;
```

Comment:
This is not a rejection case - we failed to download the zip from blob storage. It's a failure like all other and shouldn't have any special handling.